src/mixins/common-mixin.ts

```diff
@@ -43,4 +43,20 @@ export default class CommonMixin extends Vue {
   isSame (objA: {}, objB: {}, prop: string = null): boolean {
     return isEqual({ ...this.omitProp(objA, [prop]) }, { ...this.omitProp(objB, [prop]) })
   }
+
+  /**
+   * Deconstructs and sorts an array of comments
```

Comment:
The technical term for what this is doing is actually "flattening", albeit it's not flattening all levels, just one level.

Reply:
interesting, thanks! I will update this